nagios/README.md

```diff
@@ -1,32 +1,78 @@
-# Nagios Integration
+# Nagios Check
 
-## Overview
+# Overview
 
-Get metrics from nagios service in real time to:
+The Nagios check watches your Nagios server's logs and sends events to your Datadog event stream. This lets you visualize and track problems with your Nagios-monitored infrastructure with a much nicer UI—the Datadog application—than Nagios provides.
```

Comment:
The Nagios check watches your Nagios server's logs and sends events to your Datadog event stream. This is intended to help with migrations from Nagios to Datadog, but allowing you to you visualize and track problems identified in Nagios-monitored infrastructure in Datadog.